lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -35,32 +35,35 @@ def setUp(self):
         json_analytics_data = json.dumps(analytics_data)
         self.data = {"data": json_analytics_data}
 
-    @override_settings(ANALYTICS_ANSWER_DIST_URL='dummy_url')
+    @override_settings(ANALYTICS_DATA_URL='dummy_url')
+    @override_settings(ZENDESK_URL='https://stanfordonline.zendesk.com')
```

Comment:
This can be combined into the line above:
`@override_settings(ANALYTICS_DATA_URL='dummy_url', ZENDESK_URL='https://stanfordonline.zendesk.com')`